gradle/dependency-management/agp-versions.properties

```diff
@@ -1,3 +1,3 @@
 # Generated - Update by running `./gradlew updateAgpVersions`
```

Comment:
:x: as noted in the comment this file is generated, the generation task should be changed instead of changing only this file